vyper/venom/__init__.py

```diff
@@ -19,17 +19,14 @@
 
 
 def generate_assembly_experimental(
-    ctx: IRFunction, optimize: Optional[OptimizationLevel] = None
+    ctxs: tuple[IRFunction, IRFunction], optimize: Optional[OptimizationLevel] = None
```

Comment:
this should either be a list of IRFunctions or specify by names -- runtime_ir and deploy_ir.